src/components/RulesSelector.jsx

```diff
@@ -1,8 +1,8 @@
 import React, { PropTypes } from 'react';
 import RuleLabel from './RuleLabel';
 
-const makeListItems = (filters, onRuleSelected) => {
-  return Object.keys(filters)
+const makeListItems = (filters, onRuleSelected) =>
```

Comment:
I realize that, but you can still use `()` instead to indicate that it is all a block belonging to the function.